week-5/B-callbacks/exercise.js

```diff
@@ -37,15 +43,22 @@ function exercise2(result) {
   EXERCISE 3
   =======
 
-  Write a function that calls the callback with the parameter "Hello from the
+  The exercise3 function will be called for you automatically. It is called
+  with a callback function as the first parameter. Call this callback function
+  (hint: use () parentheses) with the parameter "Hello from the callback" and
```

Comment:
I like to put code syntax inside back ticks to attempt to distinguish from English syntax.